examples/counter/shared/src/lib.rs

```diff
@@ -21,15 +21,24 @@ lazy_static! {
 
 #[wasm_bindgen]
 pub fn process_event(data: &[u8]) -> Vec<u8> {
-    CORE.process_event(data)
+    match CORE.process_event(data) {
```

Comment:
Aren't these effectively just `unwrap()`?

Reply:
I think the difference is just the message you get - the errors format pretty nicely.